README.md

```diff
@@ -1,3 +1,10 @@
+[![Build Status](https://travis-ci.org/kimeunju108/alcor.svg?branch=master)](https://travis-ci.org/kimeunju108/alcor)
```

Comment:
Please use "Build status" ("status" in lower case) to be consistent with others.

How much work needs to be done to switch to official alcor account?

Reply:
The following change requires id/password of futurewei github. I could not access.
https://travis-ci.org/kimeunju108/alcor => https://travis-ci.org/futurewei-cloud/alcor